### PR TITLE
Update server_config.cfg

### PR DESCRIPTION
**Pull request by FromDarknessComesLight**
added 2x pinehills, removed 1x sandstone. also moved the other sandstone slightly in the track list.
enabled cup mode. added some basic documentation in the comments. hopefully this bullshit isn't so disorganized in WF2.

### CODE REVIEW

File: server_config.cfg [hunk 1]
```diff
@@ -526,6 +561,17 @@ el_car_class_restriction=a
...
+#pinehills raceway rally circuit
+el_add=mixed3_r3
+el_gamemode=racing
+el_laps=4
+el_bots=
+el_car_reset_disabled=0
+el_wrong_way_limiter_disabled=1
+el_car_class_restriction=a
+el_car_restriction=
+el_weather=
```
Comment (NicholasBusby) [inline server_config.cfg:573]: duplicate code

File: server_config.cfg [hunk 2]
```diff
@@ -383,6 +418,17 @@ el_car_class_restriction=a
...
+#pinehills raceway rally circuit
+el_add=mixed3_r3_rev
+el_gamemode=racing
+el_laps=4
+el_bots=
+el_car_reset_disabled=0
+el_wrong_way_limiter_disabled=1
+el_car_class_restriction=a
+el_car_restriction=
+el_weather=
```
Comment (NicholasBusby) [inline server_config.cfg:430]: why add this if not using this?

**Review by NicholasBusby — CHANGES_REQUESTED** [review]
things don't make sense

File: server_config.cfg (hunk 1 above)
Comment (FromDarknessComesLight) [reply server_config.cfg:573]: not duplicate. this adds the track going forwards, the other is the same track going backwards

File: server_config.cfg (hunk 2 above)
Comment (FromDarknessComesLight) [reply server_config.cfg:430]: not sure what you mean? I added this track to the track list. it's not a duplicate and it is not disabled. and the format is the same as the rest of the tracks

Comment (NicholasBusby) [reply server_config.cfg:430]: I don't see it being added to the loop. 
image

Comment (FromDarknessComesLight) [reply server_config.cfg:430]: the screenshotted section was a slight re-order of the tracks. I wanted to place sandstone a bit farther away from hellride so it'll potentially be skipped less. So the track list went from this:
..., deathloop, motorcity, devil's canyon, sandstone, hellride, ...
to this:
..., deathloop, sandstone, motorcity, devil's canyon, hellride, ...

**Review by NicholasBusby — APPROVED** [review]
I have the dumb while reading this at work

**Pull request merged**